### PR TITLE
Push shared policies to app collection

### PR DESCRIPTION
**Pull request by QuentinBisson**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -125,3 +125,59 @@ workflows:
           on_tag: false
           requires:
             - test-policies
+
+      - architect/push-to-app-collection:
```
Comment (MarcelMue) [inline .circleci/config.yml:129]: We do not want to push these to collections - they are included in `releases` and should therefor not be pushed to any collection :)

Comment (QuentinBisson) [reply .circleci/config.yml:129]: Well, policies-shared should be as it's not part of a release or did I not understand how you want to deploy it ?

Comment (MarcelMue) [reply .circleci/config.yml:129]: This happens when I skim and don't read properly :D

Yes you are correct here :)

Just leave out the vmware collection for now I would say.

Comment (QuentinBisson) [reply .circleci/config.yml:129]: As you wish Marcel the Genie Muller

**Review by MarcelMue — APPROVED** [review]
:rocket:

**Comment by QuentinBisson** [comment]
Thanks buddy <3

**Pull request merged**